### DISCUSSION TITLE
Connect to vaultwarden (docker) with bash script ?

### DISCUSSION THREAD

**Discussion by tamaskan** [dani-garcia/vaultwarden#3106] [Q&A]
Is there a better way (without installing the cli) or full example of this ? https://github.com/corpusops/bitwardentools/issues/14

**Comment by BlackDex** [comment]
Either use the Bitwarden CLI it self https://bitwarden.com/help/cli/, or still use the Bitwarden CLI, but then the `serve` feature, which starts a local API to interact with the vault it self https://bitwarden.com/help/cli/#serve
